tests/phpunit/tests/rest-api/rest-schema-setup.php

```diff
@@ -759,17 +759,22 @@ private function normalize_fixture( $data, $path ) {
 			return $data;
 		}
 
+		$datetime_keys = array( 'date', 'date_gmt', 'modified', 'modified_gmt' );
+
 		foreach ( $data as $key => $value ) {
-			if ( is_string( $value ) && (
-				'date' === $key ||
-				'date_gmt' === $key ||
-				'modified' === $key ||
-				'modified_gmt' === $key
-			) ) {
-				$data[ $key ] = '2017-02-14T00:00:00';
-			} else {
-				$data[ $key ] = $this->normalize_fixture( $value, "$path.$key" );
+			if ( is_string( $value ) ) {
+				if ( in_array( $key, $datetime_keys, true ) ) {
+					$data[ $key ] = '2017-02-14T00:00:00';
+					continue;
+				}
+
+				if ( 1 === preg_match( '/^post-\d+$/', $value ) ) {
+					$data[ $key ] = 'post-1073';
```

Comment:
What is this? Why is this hardcoded? Is it for test stability? Might be good to comment.

Reply:
Yes, it's to ensure test stability. Fixed in d24ee393411efe9f87ab5e51d2ea6878d7cecc7e.
P.S. I'll quckly check if hardcoding the value can be avoided. For now, let's go with the comment.

Reply:
I've found a better way of adding these fixture replacements.
The current implementation should work, but there's a better approach that's outlined here: https://core.trac.wordpress.org/ticket/61369
Hopefully, somebody (a mantainer?) can take a look when they have time. The current implementation is also fine in a sense that it should work for the purposes of mocking the REST API, but is not ideal (so, not an urgent matter).